tiny_dnn/core/kernels/global_avepool_op_avx.h

```diff
@@ -0,0 +1,83 @@
+/*
+    Copyright (c) 2017, Taiga Nomi
+    All rights reserved.
+
+    Use of this source code is governed by a BSD-style license that can be found
+    in the LICENSE file.
+*/
+#pragma once
+
+namespace tiny_dnn {
+namespace kernels {
+
+#ifdef CNN_USE_AVX
```

Comment:
I rectified this one - similar to what is done in conv2d and fully connected kernels.